guide/10-mapping-and-visualization-alt/part7_utility_functions_for_the_map_widget.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>The <code>set_js_cdn</code> function is called before the creation of any <code>MapView</code> object, and each instantiated object will use the specified <code>js_cdn</code> parameter as the ArcGIS API for JavaScript <code>CDN</code> URL, rather than the default <code>http://js.arcgis.com/4.X/</code>. This functionality is necessary in disconnected environments if the portal you are connecting to does not ship with the minimum necessary JavaScript API version. </p><p> </p><p><strong>Note:</strong> You may not need to call this function to view the widget in disconnected environments. For instance, if your computer cannot reach <code>js.arcgis.com</code> and you have a <code>GIS</code> connection to a portal, the widget will automatically attempt to use that Portal’s JS API that it ships with. </p><p> </p><p>For example, if <code>gis.map()</code> does not return a map, run the following code to configure the map widget with the Javascript API shipped with the portal:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>"if the enterprise you are connecting " -- capitalize E in Enterprise (since its a product)</p>